CondFormats/RPCObjects/interface/RPCInverseAMCLinkMap.h

```diff
@@ -0,0 +1,34 @@
+#ifndef CondFormats_RPCObjects_RPCInverseAMCLinkMap_h
```

Comment:
For CondFormats that are serialized, I was advised against using typedefs for containers after running into problems.  These are dependent records however, but I implemented it this way to stay in line with the corresponding non-inverse classes.

I haven't tested if the macro's involved with these records give problems - but even if they don't I should first ask if it's supported.  Do you prefer if I try and change it?